controllers/contacts.js

```diff
@@ -0,0 +1,77 @@
+const { Contact } = require('../contacts.schema');
+
+const listContacts = async () => {
+  try {
+    const contacts = await Contact.find();
+    return contacts;
+  } catch (error) {
+    console.error('Error:', error.message);
```

Comment:
```suggestion
    console.error('Error:', error);
```
Dobrym pomysłem może też być robienie console całego błędu, bo message czasami niewiele pokazuje a cały obiekt pokazuje również miejsca gdzie jest błąd itp. Polecam, tam gdzie robisz console.log błędu.